p/riscv_test.h

```diff
@@ -259,8 +260,8 @@ reset_vector:                                                           \
 #define RVTEST_DATA_BEGIN                                               \
         EXTRA_DATA                                                      \
         .pushsection .tohost,"aw",@progbits;                            \
-        .align 6; .global tohost; tohost: .dword 0;                     \
-        .align 6; .global fromhost; fromhost: .dword 0;                 \
+        .align 8; .global tohost; tohost: .dword 0; .size tohost, 8;    \
+        .align 8; .global fromhost; fromhost: .dword 0; .size fromhost, 8;\
```

Comment:
Why 256 bytes of allignment?

Reply:
Sorry, Changing the allignment to 8 may be not neccessary. What really works is setting the '.size' of tohost and fromhost:
```
    if (strcmp("fromhost", st_name) == 0) {
        address_symbol_set |= 1;
        fromhost_addr = st_value;
        if (st_size != 8) {
            error_report("HTIF fromhost must be 8 bytes");
            exit(1);
        }
    } else if (strcmp("tohost", st_name) == 0) {
        address_symbol_set |= 2;
        tohost_addr = st_value;
        if (st_size != 8) {
            error_report("HTIF tohost must be 8 bytes");
            exit(1);
        }
    } 
```

Reply:
Ok, let’s get rid of it, then.